src/Overlay.tsx

```diff
@@ -166,7 +166,7 @@ function Overlay({
   ...outerProps
 }: OverlayProps) {
   const popperRef = useRef({});
-  const [ref, marginModifiers] = usePopperMarginModifiers();
+  const [ref, modifiers] = useOverlayOffset();
```

Comment:
hmm this API is unfortunate...ideally you'd want the the Popover component itself to specify this, might be feasible with context, but i'd be afraid of timing issues. Lets make sure we aren't calling this hook "public" in case we want to remove it later